daemon-tests/tests/happy_path.rs

```diff
@@ -616,14 +616,16 @@ async fn taker_notices_lack_of_maker() {
     )
     .await;
 
+    sleep(Duration::from_secs(5)).await;
+
     assert_eq!(
         ConnectionStatus::Online,
         next(taker.maker_status_feed()).await.unwrap()
     );
 
     drop(maker);
 
-    sleep(taker_config.heartbeat_interval).await;
+    sleep(Duration::from_secs(5)).await;
```

Comment:
Same here, I think a short comment could help understand why we need to sleep exactly 5 secs.